File: CS-Examples/03_POP3 Client/WorkingWithMessages/GetMailInfo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Email;
using Spire.Email.Pop3;

namespace GetMailInfo
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            // Create a POP3 client
            Pop3Client client = new Pop3Client();
            client.Host = this.HostInput.Text;
            client.Username = this.Username.Text;
            client.Password = this.Password.Text;
            client.Port = int.Parse(this.PortInput.Text);
            client.EnableSsl = true;
            client.Connect();
            Pop3MessageInfoCollection msgs = client.GetAllMessages();
            for (int i = 0; i < msgs.Count; i++)
            {
                MessageBox.Show(string.Format("SequenceNumber:{0};Size:{1};UniqueId:{2}", msgs[i].SequenceNumber, msgs[i].Size, msgs[i].UniqueId));
            }
        }
    }
}

File: VB-Examples/03_POP3 Client/WorkingWithMessages/GetMailInfo.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Email
Imports Spire.Email.Pop3

Namespace GetMailInfo
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			' Create a POP3 client
			Dim client As New Pop3Client()
			client.Host = Me.HostInput.Text
			client.Username = Me.Username.Text
			client.Password = Me.Password.Text
			client.Port = Integer.Parse(Me.PortInput.Text)
			client.EnableSsl = True
			client.Connect()
			Dim msgs As Pop3MessageInfoCollection = client.GetAllMessages()
			For i As Integer = 0 To msgs.Count - 1
                MessageBox.Show(String.Format("SequenceNumber:{0};Size:{1};UniqueId:{2}", msgs(i).SequenceNumber, msgs(i).Size, msgs(i).UniqueId))
			Next i
		End Sub
	End Class
End Namespace